### PR TITLE
Improved Error handling in Xworkflow1 and Xworkflow2

### PR DESCRIPTION
**Pull request by SyedMurtazaHassan46**
# XkOps

## Issue

[Issue no 65](https://github.com/XgridInc/xkops/issues/65)

## Description

Added and improved error handling in Xworkflow 1 and Xworkflow 2 to include proper try-except section and codes

## Checklist

- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [ ] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes (if applicable)
- [x] Every function, interface, class has a comment describing what it does and input/output parameters

### CODE REVIEW

**Review by umerxgrid — APPROVED** [review]
+1

File: xWorkflows/xWorkflow_2/flask_backend/app.py [hunk 1]
```diff
@@ -8,28 +8,41 @@
 
 # Connect to MongoDB
 client = MongoClient("mongodb://admin:password@mongo-db-0.mongo-db.default.svc.cluster.local:27017")
```
Comment (HamzaXgrid) [inline xWorkflows/xWorkflow_2/flask_backend/app.py:10]: Pass username and password using env

Comment (SyedMurtazaHassan46) [reply xWorkflows/xWorkflow_2/flask_backend/app.py:10]: There is a separate ticket already created for this, in this ticket I worked on error handling. The PR for this code was already merged so we can address this in the env and secrets ticket.

**Review by HamzaXgrid — APPROVED** [review]
+1

**Pull request merged**